framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -263,7 +263,7 @@ EigenExecutionerBase::inversePowerIteration(unsigned int min_iter,
 
     // save the initial residual
     if (iter == 0)
-      initial_res = _eigen_sys._initial_residual_before_preset_bcs;
+      initial_res = _eigen_sys.referenceResidual();
```

Comment:
This is a real change, and I think this is a right change.

This could potentially lead to some diffs in theory, but I am not seeing any.